packages/styled-components/src/sheet/Sheet.js

```diff
@@ -22,6 +22,8 @@ class StyleSheet implements Sheet {
 
   tag: void | GroupedTag;
 
+  useCSSOM: boolean;
```

Comment:
Lets maybe make this part of the constructor and refactor it to take a prams object?

Are we also calling this CSSOM? We could continue to call it speedy injection or use "useLegacyInjection" or sth

Reply:
I definitely think calling it CSSOM is more obvious... "speedy" is cute but an unofficial name and not helpful if people are trying to google for our usage of it you know?

Reply:
> Lets maybe make this part of the constructor and refactor it to take a prams object?

It'd break backward compat. We can look into that for v6 though.

Reply:
Will it though? The Stylesheet was a secret internal and I'd expect it to break anyway since most people will use it for the old static reset method.

I do think it's reasonable to break backwards compatibility here. Jest- styled-components will need to be changed anyway

Reply:
It's not really a secret internal, we allow people to make their own sheet to supply to `StyleSheetManager` in v3+.

Reply:
Ok nevermind it's actually in that spooky ghosts object so I guess it is. Alrighty.